apps/src/StudioApp.js

```diff
@@ -1811,6 +1811,25 @@ StudioApp.prototype.handleEditCode_ = function (config) {
   var aceEditor = this.editor.aceEditor;
   aceEditor.session.setMode('ace/mode/javascript_codeorg');
 
+  // Extend the command list on the ace Autocomplete object to include the period:
+  var Autocomplete = window.ace.require("ace/autocomplete").Autocomplete;
+  Autocomplete.prototype.commands['.'] = function(editor) {
```

Comment:
Will you point me to Ace docs for this, or help me understand when this code runs?  I'm a little confused by where `editor.complete.completions.filtered` comes from, or why we only continue showing the autocomplete menu when it contains an exact match, or what `exactMatch` means here, exactly.